src/components/network/network-table.js

```diff
@@ -94,6 +112,37 @@ const NetworkTable = (props) => {
         [lineEdit, tabIndex]
     );
 
+    useEffect(() => {
+        fetchConfigParameter(
```

Comment:
why fetch the parameters when changing tab ? Everything has already been fetched at page load and stored in redux, and the websocket connection updates redux when there is a change, so you can just use them direclty no?

Reply:
See comment below